api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -0,0 +1,116 @@
+  <style>
+    body {
+      background-color: #F1F1F1;
+      padding-left: 1rem;
+      padding-top: 1rem;
+      width: 600px;
+    }
+    * {
+      font-family: "Gotham SSm A", "Gotham SSm B", sans-serif;
+    }
+    .welcome-message {
+      margin-top: 2rem;
+      margin-bottom: 1rem;
+      color: #302C71;
+      font-size: 30px;
+      font-weight: 300;
+      line-height: 36px;
+    }
+
+    .text {
+      color: rgba(0,0,0,0.87);
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 22px;
+      margin-bottom: 1rem;
+    }
+
+    .text.user-field {
+      width: 8rem;
+      display: inline-block;
+      margin-bottom: 0;
+    }
+    .user-info {
+      font-weight: bold;
+      display: inline-block;
+    }
+    .card {
+      height: 70px;
+      width: 560px;
+      border-radius: 4px;
+      background-color: #FFFFFF;
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: flex-start;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet {
+      height: 31px;
+      margin-left: 1rem;
+      margin-right: 1rem;
+      width: 31px;
+      border-radius: 15px;
+      background-color: #302C71;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      color: #FFFFFF;
+    }
+
+    .card-text {
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+      width: 460px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://myaccount.google.com/signinoptions/two-step-verification">
+    <div class="bullet">1</div>
+    <div class="card-text"><div class="link">Sign into google</div> with your new account, change your password, and set up 2-Step Verification</div>
+  </a>
+  <a class="card" href="https://myaccount.google.com/intro/recovery/email?hl=en-US">
+    <div class="bullet">2</div>
+    <div class="card-text"><div class="link">Add</div> your recovery email address</div>
```

Comment:
Doesn't Google prompt you to do this when you login, if you haven't already set it up? If so I'd suggest we just drop this.

Reply:
I haven't seen it when I do it incognito. It may be different on accessing a security page.